server/controllers/transactions.js

```diff
@@ -2,7 +2,8 @@ const { Transaction, Wallet, TokenType } = require('../models');
 const nacl = require("tweetnacl");
 const StellarSdk = require("stellar-sdk");
 const dec = require("../utils/dec");
-const TOKEN_GRAVEYARD_ADDRESS = process.env.TOKEN_GRAVEYARD_ADDRESS;
```

Comment:
why this change?

Reply:
No reason to store in ENV if we can just have a constant at the top of a file, publicly exposing it

Reply:
right but what if we want to change it? would require a whole PR, merge, build, upload etc